src/arm_mobility/launch/robot_bringup.launch.py

```diff
@@ -0,0 +1,119 @@
+import os
+from launch import LaunchDescription
+from launch.actions import ExecuteProcess
+from launch_ros.actions import Node
+from ament_index_python.packages import get_package_share_directory
+from moveit_configs_utils import MoveItConfigsBuilder
+
+
+def generate_launch_description():
+
+    description_arguments = {
+        "robot_ip": "xxx.yyy.zzz.www",
+        "use_fake_hardware": "false",
+        "gripper": "robotiq_2f_85",
+        "dof": "6",
+    }
+
+    # planning_context
+    moveit_config = (MoveItConfigsBuilder("gen3", package_name="kinova_gen3_6dof_robotiq_2f_85_moveit_config")
+        .robot_description(mappings=description_arguments)
+        .planning_pipelines(pipelines=["ompl", "chomp", "pilz_industrial_motion_planner"])
+        .to_moveit_configs()
+    )
+
+    # Load  ExecuteTaskSolutionCapability so we can execute found solutions in simulation
+    move_group_capabilities = {
+        "capabilities": "move_group/ExecuteTaskSolutionCapability"
+    }
+    # not sure what ^this is doing.
```

Comment:
nit: maybe make it a TODO so you can come back to it if you may need to make changes

Reply:
Done.